### PR TITLE
APPLICATION FIX

### PR DESCRIPTION
**Pull request by rokikon**
Upgraded Angular to RC5 and removed StoreDevTools as it is not supported yet

### CODE REVIEW

File: src/build-apps/app/tsconfig.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "target": "es5",
     "files": "./main.ts"
   },
-  "exclude": ["node_modules", "test.ts"],
+  "exclude": ["node_modules", "test.ts", "./build-apps"],
```
Comment (agnislav) [inline src/build-apps/app/tsconfig.json]: may be not needed after my last PR #176

**Comment by agnislav** [comment]
Is this the `'.' module` error fix?

**Comment by rokikon** [comment]
@agnislav nope, i've had another issue) But now everything is ok

File: src/build-apps/app/tsconfig.json (hunk 1 above)
Comment (rokikon) [inline src/build-apps/app/tsconfig.json]: @agnislav well, anyway we need to exclude this directory from compilation flow and you will rebase your PR after #178) So why not?)

Comment (agnislav) [inline src/build-apps/app/tsconfig.json]: np :)

**Pull request merged**